src/V3EmitCSyms.cpp

```diff
@@ -296,10 +319,11 @@ class EmitCSyms final : EmitCBaseVisitor {
             const string type
                 = (nodep->origModName() == "__BEGIN__") ? "SCOPE_OTHER" : "SCOPE_MODULE";
             const string name = nodep->scopep()->shortName() + "__DOT__" + nodep->name();
-            const string name_dedot = AstNode::dedotName(name);
+            const string name_pretty = AstNode::vpiName(name);
             const int timeunit = m_modp->timeunit().powerOfTen();
-            m_vpiScopeCandidates.insert(
-                std::make_pair(name, ScopeData(scopeSymString(name), name_dedot, timeunit, type)));
+            m_vpiScopeCandidates.insert(std::make_pair(
+                name, ScopeData(scopeSymString(name), name_pretty, timeunit, type,
+                                type == "SCOPE_MODULE" ? nodep->origModName() : "")));
```

Comment:
```suggestion
                name, ScopeData{scopeSymString(name), name_pretty, timeunit, type,
                                type == "SCOPE_MODULE" ? nodep->origModName() : ""}));
```

Might need some other minor changes when upgrade to {}.